pkg/rke/k8s_rke_system_images.go

```diff
@@ -9342,6 +9342,51 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.20",
 		},
+
+		// Enabled in Rancher v2.7.x
+		"v1.27.2-rancher2-1": {
+			Etcd:                      "quay.io/coreos/etcd:v3.5.7",
+			Kubernetes:                "rayandas/hyperkube:v1.27.2-rancher2",
+			Alpine:                    "rancher/rke-tools:v0.1.89",
+			NginxProxy:                "rancher/rke-tools:v0.1.89",
+			CertDownloader:            "rancher/rke-tools:v0.1.89",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.89",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.22.20",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.22.20",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.22.20",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			Flannel:                   "rancher/mirrored-flannel-flannel:v0.21.4",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher8",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.25.0",
```

Comment:
calico has a new version out, v3.26.1 so we'd need to update with introducing new minor v1.27